website/docs/explorer/getting-started.mdx

```diff
@@ -35,7 +35,20 @@ metadata:
 spec:
   # ... other spec components
   values:
-    enableExplorer: true
+  explorer:
+    enabled: true
+    collector:
+      # ServiceAccount that explorer will use to watch clusters for resources
+      serviceAccount:
+        name: "collector"
+        namespace: "flux-system"
+    cleaner:
+      disabled: false
+    enabledFor:
+      - applications
+      - sources
+      - gitopssets
+      - templates
```

Comment:
it feels that we should expand with comments to provide guidance to the user on how use these flags. for example: the user could think that is configuration for deciding which type of data to collect and not for the UI only